src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7406,4 +7406,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_ParamsArrayInLambdaOnly_Title" xml:space="preserve">
     <value>Parameter has params modifier in lambda but not in target delegate type.</value>
   </data>
+  <data name="IDS_DisableLengthBasedSwitch" xml:space="preserve">
+    <value>disable-length-based switch</value>
```

Comment:
>disable-length-based [](http://example.com/codeflow?start=11&length=20)

Space instead of the first hyphen?

Reply:
Actually intended for hyphens throughout. Fixed and added a command-line test.